### PR TITLE
Eslint configuration

### PR DESCRIPTION
**Pull request by ramakotaiah12**

### CODE REVIEW

File: packages/tech-admin/package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
   "devDependencies": {
     "@types/react": "^17.0.3",
     "@types/react-dom": "^17.0.3",
+    "eslint": "^7.24.0",
```
Comment (punith77) [inline packages/tech-admin/package.json]: Do we need eslint in each package?  I think we can configure it on top-level. Please check

File: packages/tech-provider/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (punith77) [inline packages/tech-provider/.eslintrc.json]: Seems this is the duplicate code 
Try to configure it on top-level - in that case, we don't need duplicate code

File: packages/package.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (punith77) [inline packages/package.json]: There are 3 commits to this PR.
Please make one commit per PR - squash commits to 1

File: package.json [hunk 4]
```diff
@@ -10,6 +10,10 @@
     "packages/*"
   ],
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^4.22.0",
+    "@typescript-eslint/parser": "^4.22.0",
+    "eslint": "^7.24.0",
```
Comment (punith77) [inline package.json:16]: do we need both eslint and @typescript-eslint?

File: packages/package.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+  "name": "packages",
+  "version": "1.0.0",
```
Comment (punith77) [inline packages/package.json]: we dont need this package.json. Any specific reason for adding this?

File: packages/tech-components/index.tsx [hunk 6]
```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
 
 
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```
Comment (punith77) [inline packages/tech-components/index.tsx]: You need to add type to remove the error(something like React.Node). Avoid quick fixes for eslint in most cases

File: packages/tech-user/src/index.tsx [hunk 7]
```diff
@@ -1,21 +1,22 @@
...
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
 
-import ComponentLibrary from 'tech-components'
+import ComponentLibrary from 'tech-components';
 
 export interface HelloProps { name: string; }
 
 
+// eslint-disable-next-line @typescript-eslint/ban-types
```
Comment (punith77) [inline packages/tech-user/src/index.tsx]: same thing here - add type

File: package.json (hunk 4 above)
Comment (ramakotaiah12) [reply package.json:16]: Yes, we need both. Eslint does not understand typescript syntax without plug-in and parser.

File: packages/package.json (hunk 5 above)
Comment (ramakotaiah12) [reply packages/package.json]: No, we don't need package,json in packages, and removed.

File: packages/package.json [hunk 8]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "packages",
```
Comment (punith77) [inline packages/package.json]: I'm still seeing package.json in packages. Did you push your changes?

**Review by punith77 — APPROVED** [review]

**Pull request merged**